DiscordChatExporter.Cli/Commands/ExportChannelsCommand.cs

```diff
@@ -19,9 +26,86 @@ public class ExportChannelsCommand : ExportCommandBase
     )]
     public required IReadOnlyList<Snowflake> ChannelIds { get; init; }
 
+    [CommandOption(
+        "include-threads",
+        Description = "Which types of threads should be included.",
+        Converter = typeof(ThreadInclusionModeBindingConverter)
+    )]
+    public ThreadInclusionMode ThreadInclusionMode { get; init; } = ThreadInclusionMode.None;
+
     public override async ValueTask ExecuteAsync(IConsole console)
```

Comment:
Why not perform this in the `ExportCommandBase` instead? Currently, that's where we unwrap categories already.

Reply:
Wasnt sure if it made sense to be in all children but looks like they all handle it in some way.  moved it.

Reply:
Oh I meant also moving the unwrapping logic from `ExportChannelsCommand.ExportAsync(...)` to `ExportCommandBase.ExportAsync(...)`

Reply:
Added now.